gulpfile.js

```diff
@@ -11,7 +11,11 @@ gulp.task('verify', function() {
 });
 
 gulp.task('test', function() {
-	obt.test.npmTest(gulp);
+	obt.test.npmTest(gulp)
```

Comment:
You can return a Promise or stream within `gulp.task`, would this set up error handling for you? https://github.com/gulpjs/gulp/blob/master/docs/API.md#return-a-promise